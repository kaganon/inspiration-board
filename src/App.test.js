import React from 'react';
import ReactDOM from 'react-dom';
import { mount, shallow } from 'enzyme';
import App from './App';

describe('App', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

});

describe('<App />', () => {
  test('that it renders App with shallow rendering', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });

  test('will match the last snapshot with deep rendering', () => {
    const wrapper = mount(<App />);
    expect(wrapper).toMatchSnapshot;

    wrapper.unmount();
  });
  
});
