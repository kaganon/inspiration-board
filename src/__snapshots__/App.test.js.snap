// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> that it renders App with shallow rendering 1`] = `
<section>
  <header
    className="header"
  >
    <h1
      className="header__h1"
    >
      <span
        className="header__text"
      >
        Inspiration Board
      </span>
    </h1>
  </header>
  <Board
    boardName="Ada-Lovelace"
    url="https://inspiration-board.herokuapp.com"
  />
</section>
`;
